pkg/services/image/event_image_iso_requested.go

```diff
@@ -45,13 +40,23 @@ func (ev EventImageISORequestedBuildHandler) Consume(ctx context.Context) {
 		eventlog.Error("Error unmarshaling the image")
 		return
 	}
-	log := eventlog.WithFields(log.Fields{
+	if image.OrgID == "" || image.RequestID == "" {
+		eventlog.WithFields(log.Fields{
+			"message":   "Malformed image request, vital data missing",
```

Comment:
instead, call vital, could we change to required? wdyt?
```suggestion
			"message":   "Malformed image request, required data missing",
```